donate.md

```diff
@@ -1,22 +1,26 @@
 ---
 layout: page
 ---
-Our mission is to **empower people** with technology through 
-**teaching and facilitating access**, enlarging the community of people who 
+Our mission is to **empower people** with technology through
+**teaching and facilitating access**, enlarging the community of people who
 **give back and teach others**.
 
 Contributions to Bridge Foundry allow thousands of volunteers across the world
-to teach workshops and provide resource to women and others who are underserved 
-in access to technology and tech skills. 
+to teach workshops and provide resource to women and others who are underserved
+in access to technology and tech skills.
 
-Most workshops happen without needing any funds with companies providing 
-space and food for our events. Providing food is an equalizer, making it so people who can't 
+Most workshops happen without needing any funds with companies providing
+space and food for our events. Providing food is an equalizer, making it so people who can't
 afford to go out for lunch are not separated from those that can. Meals also
 provide a comfortable space for informally networking.  
 
 Sometimes the only funds that a volunteer organizer needs is $50
-to pay a security guard or $500 for a plane ticket to fly a teacher 
-to a new community. 
+to pay a security guard or $500 for a plane ticket to fly a teacher
+to a new community.
+<br>
```

Comment:
Wanted to add more space before and after button. Is this the best way to do it?